src/OpenTelemetry/Context/Propagation/B3Format.cs

```diff
@@ -76,7 +77,7 @@ public SpanContext Extract<T>(T carrier, Func<T, string, IEnumerable<string>> ge
                 }
                 else
                 {
-                    throw new SpanContextParseException("Missing X_B3_TRACE_ID.");
+                    return SpanContext.BlankRemote;
```

Comment:
Same, could we emit log when failing to parse span context is invalid?

Reply:
This branch represents the case where there is no header in the message with b3 (this is a happy and default case). I don't think any logging needs to happen here.